src/scripts/main.js

```diff
@@ -1,3 +1,13 @@
 'use strict';
 
-// write code here
+function sortList(list) {
+  let myList = list;
+
+  myList = document.querySelector('ul');
+
+  const salaryElement = myList.querySelector('data-salary');
+
+  return salaryElement;
```

Comment:
Returning `salaryElement` does not contribute to sorting the list. Consider implementing a sorting algorithm or logic here.